src/Extractor/BaseExtractor.php

```diff
@@ -127,15 +127,6 @@ protected function processExportResult(ExportConfig $exportConfig, ?string $maxV
         return $output;
     }
 
-    protected function connect(): void
-    {
-        try {
-            $this->createConnection($this->databaseConfig);
-        } catch (Throwable $e) {
-            throw new UserException('Error connecting to DB: ' . $e->getMessage(), 0, $e);
```

Comment:
This code is in `db-extractor-adapter`: https://github.com/keboola/db-extractor-adapter/blob/117ee22e25c8cbdcd80cf8f830b6966bd21a2017/src/Connection/BaseDbConnection.php#L111

I found out when writing RetryTest that there is a duplicate prefix.